docs/yaml_docs/pipeline_clustering_yml.md

```diff
@@ -0,0 +1,276 @@
+            Can specify an array: 0.25,0.5,0.8
+
+## Parameters for clustering 
+
+  - <span class="parameter">clusterspecs:</span>
+      - <span class="parameter">rna:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+      - <span class="parameter">prot:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden.
+
+      - <span class="parameter">atac:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+      - <span class="parameter">multimmodal:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.5, 0.7<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1 
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden.
+
+      - <span class="parameter">spatial:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1 
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+
+## Parameters for finding marker genes 
+
+In this part of the analysis we define parameters to run marker analysis. 
+By default, pseudo_seurat is set to False, and we run [scanpy.tl.rank_genes_groups](https://scanpy.readthedocs.io/en/stable/generated/scanpy.tl.rank_genes_groups.html). 
+When pseudo_seurat is set to True then a [python implementation](https://github.com/DendrouLab/panpipes/blob/main/panpipes/python_scripts/run_find_markers_multi.py) of `Seurat:::FindMarkers` is run
+
+  - <span class="parameter">markerspecs:</span> <br>
+     - <span class="parameter">rna:</span><br>
+       - <span class="parameter">run </span> `Boolean`, Default: True<br>
+       - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+         Which layer stores counts for differential expression test.
+       - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+       Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+       - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
+       Minimal number of cells in a cluster. If the cluster contains less than this number of cells, the marker analysis won't be run.
```

Comment:
I would say " Marker analysis  is run for clusters  >= mincells. If a cluster ncells < mincells , then the cluster is excluded from marker analysis